src/backend/requirements.in

```diff
@@ -36,14 +37,16 @@ dulwich                                 # pure Python git integration
 docutils                                # Documentation utilities for auto admin docs
 drf-spectacular                         # DRF API documentation
 feedparser                              # RSS newsfeed parser
-gunicorn                                # Gunicorn web server
+gunicorn>=22.0.0                        # Gunicorn web server
+invoke>=2.2.0                           # Invoke build tool
 pdf2image                               # PDF to image conversion
 pillow                                  # Image manipulation
 pint                                    # Unit conversion
 pip-licenses                            # License information for installed packages
 ppf.datamatrix                          # Data Matrix barcode generator
 python-barcode[images]                  # Barcode generator
 python-dotenv                           # Environment variable management
+python-ldap                             # LDAP integration
```

Comment:
This blows up the system requirements for installing the package on Debian - why was this changed?

Reply:
So does the installer not ship with ldap support by default? Happy to revert this, if so.

Reply:
No, ldap requires a lot of local dependencies.